tensorflow_serving/model_servers/model_service_impl.cc

```diff
@@ -17,20 +17,54 @@ limitations under the License.
 
 #include "tensorflow_serving/model_servers/get_model_status_impl.h"
 #include "tensorflow_serving/model_servers/grpc_status_util.h"
+#include "tensorflow_serving/util/status_util.h"
 
 namespace tensorflow {
-namespace serving {
-
-::grpc::Status ModelServiceImpl::GetModelStatus(
-    ::grpc::ServerContext* context, const GetModelStatusRequest* request,
-    GetModelStatusResponse* response) {
-  const ::grpc::Status status = tensorflow::serving::ToGRPCStatus(
-      GetModelStatusImpl::GetModelStatus(core_, *request, response));
-  if (!status.ok()) {
-    VLOG(1) << "GetModelStatus failed: " << status.error_message();
-  }
-  return status;
-}
-
-}  // namespace serving
+  namespace serving {
+
+    ::grpc::Status ModelServiceImpl::GetModelStatus(::grpc::ServerContext *context,
```

Comment:
It would be better to leave the unchanged lines alone (not re-format them), so it's more clear which lines you actually changed in your PR. Thanks.